views/about.hbs

```diff
@@ -0,0 +1,14 @@
+<link rel="stylesheet" href="../styles/style.css" />
+
+{{>layout}}
+
+<div class="container">
+  <img src="/images/photo16.jpg" />
+  <p class="abouttext">
+    Hi, my name is Susan. I was born in Florida, raised in Paris but I'm
+    100% Persian. <br /><br />
```

Comment:
Adding br tags is considered 'bad', it's better to style things with css rather than adding tags